### ISSUE TITLE
[SST2 + NextjsSite] Noticed a substantial Next.js SSR perfomance hit after upgrading to v2

### ISSUE THREAD

**Issue by igormartimiano** [sst/sst#2498]
After upgrading my Nextjs site to use SST v2 I noticed a substantial hit on SSR response times and size. I believe it might be directly related to open-next. Here is some quick comparisons between v1 and v2 on the same page:

|v2|v1|
|---|---|
![image](https://user-images.githubusercontent.com/22182599/219173174-6e53a13c-5e23-4905-a620-7e0d79a14515.png) | ![image](https://user-images.githubusercontent.com/22182599/219172272-b4b425b7-9e9a-4691-9ff9-3b3a3fe60b83.png)

I know `open-next` is really new and I understand that it might have issues like this. Also `serverless-next.js` is unmaintained, as I'm coming from it. This perfomance hit is quite alarming in my specific case and I'll probably have to downgrade to use SST at all. Sadly I also don't have the time to go deeper on this issue 😕

Would really appreciate any comments around it!

**Comment by khuezy** [comment]
Yikes, why aren't the requests running in parallel? There's some optimization on open-next: https://github.com/serverless-stack/open-next/pulls.

Can you explain exactly what your site is doing to behave like that?

**Comment by fwang** [comment]
@igormartimiano are these cold start times? If you refresh a few times, do the requests still take this long?

**Comment by igormartimiano** [comment]
@fwang The first and last one on v2 are cold starts if I remember correctly. On v1, only the first is. 
The others I did as you said, kept refreshing them after they were fully loaded and these are the results.

**Comment by igormartimiano** [comment]
@khuezy I ran them one after another on purpose, wanted to get an average response time for a specific page. 
Just saw your PR on open-next, hope it helps!

Not sure what kind of explanation around my site would be useful, but I'm willing to share anything that could be of any help. I didn't notice this behavior on spamming requests to the SSR response, if anything I noticed the response time lowered a bit in each repeated request until they kind of hit a plateau.
